1_fetch/src/get_site_data.R

```diff
@@ -13,14 +14,14 @@ get_site_data <- function(sites_info, state, parameter) {
 
   # actually pull the data
   site_data <- dataRetrieval::readNWISdv(
-    siteNumbers=site_info$site_no, parameterCd=parameter) %>%
+    siteNumbers=sites_info$site_no, parameterCd=parameter) %>%
```

Comment:
I think it would be easier to read this code if instead of changing all of the instances of `site_info` to `sites_info` to account for having taken out the `site_info <- filter(sites_info, state_cd == state)` line, you changed the function definition to take a parameter named `site_info`, e.g. `get_site_data <- function(site_info, state, parameter)`.

Reply:
true that is easier...haha. But I guess since the change is already made, I'll leave as is.